### DISCUSSION TITLE
Ability to request account deletion for inactive accounts

### DISCUSSION THREAD

**Discussion by ghost** [community/community#30719] [Other Feature Feedback, Questions, & Ideas]
So there are lots of accounts that litter GitHub with almost 0 contributions (and sometimes put questionable things on their Pages site) that often infringes people's rights. Such accounts are:

- `maindotc`
- `xrch`
- `charles`
- `turtle`
- `gh`

It would be nice on such accounts (that had 0 activity in the last 6 - 18 months, you decide) a message will display:

```
This account hasn't been active. Would you like to request this account be deleted? (button: Request)
```

And when the user clicks `Request` a form will show up with the usual yada yada. It would also be great if we could automatically set a organization/profile name if the request goes through.

**Comment by victorlim4** [comment]
@0xC4aE1e5 - This functionality would be amazing!, the only problem is that it makes room to take down the account of some project that uses github as a backup of projects, or for any other reason. I think it would be interesting to extend the downtime and create a set of requirements to release this functionality. Requirements like the one you quoted, "(and sometimes put questionable things on their Pages site)".

**Comment by victorlim4** [comment]
There is already a similar solution on github enterprise, here the documentation link:
[Managing Dormant Users](https://docs.github.com/en/github-ae@latest/admin/user-management/managing-users-in-your-enterprise/managing-dormant-users)

 If interested, I suggest opening an issue in the github repository. What you think about this @github-staff?

**Comment by radar-blip** [comment]
@0xC4aE1e5 ,
Some people keep their accounts private and dont show contributions.

**Reply by victorlim4** [reply]
Exactly, there's this factor too

**Reply by radar-blip** [reply]
Yes, It's been discussed before.

**Reply by radar-blip** [reply]
So, just because you can't see any activity, doesn't mean it's not being used. Thats basically what was dicussed before. This was coming from a staff member if I remember correctly.
